guides/common/modules/ref_capsule-server-scalability-considerations.adoc

```diff
@@ -4,7 +4,12 @@
 = {SmartProxyServer} Scalability Considerations
 
 The maximum number of {SmartProxyServer}s that {ProjectServer} can support has no fixed limit.
-The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer} with {RHEL} 7.
+ifndef::satellite[]
+The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer} with {EL} 8 or {EL} 7.
```

Comment:
I'm cautious to make this claim. Did we really test it on EL8?

Reply:
@mjivraja We checked (in email) that we didn't test it on RHEL8. So for now, let's keep the line for non-sat and RHEL7 and remove it for Sat below.

Reply:
Since I was on the CC: the performance team didn't observe significant changes between EL7 & EL8 so I'd be ok with changing the claim to EL8 now. Then we should ask the performance team to do the real test. There's a good chance we'll need to revisit the section after with their new findings.

Reply:
Alright. I am right now moving with @Lennonka's suggestion. Can modify further after we get the findings.